### ISSUE TITLE
Substance relationship table updates

### ISSUE THREAD

**Issue by cmgrulke** [Chemical-Curation/chemcurator_vuejs#111]
**As a** *curator*
**I want** *to be able to edit on a single row basis the content in the substance table*
**So that** *I can correct the relationships shown in the table*

**Acceptance Criteria:**
In the substance table
- I can edit the qc_note
- I can select a different source from a pull down menu of all soures)
- I can change the relationship using a pull down menu contain all "forward" and "backward" relationships
- I can change the dtxsid
- if on submit there is an error, I am notified

**Comment by michael-barbour** [comment]
#65 adds a table to the Substance page for Substance Relationships.

**Comment by cmgrulke** [comment]
Yup, 65 did exactly replicate what I was asking for here.  I think we should just delete this one and chalk it up to PO failure.

**Comment by cmgrulke** [comment]
I decided to alter this ticket to be the update ticket

**Comment by jtumkur** [comment]
Currently the table is Read only.
Directed graph by putting the substance id.
Directed relationship - Forward and Reverse relationship.
2 labels on a single relationship type.
